src/broadcast_stage.rs

```diff
@@ -183,38 +187,29 @@ impl BroadcastStage {
         window: &SharedWindow,
         entry_height: u64,
         receiver: &Receiver<Vec<Entry>>,
+        leader_scheduler_option: &Option<Arc<RwLock<LeaderScheduler>>>,
```

Comment:
is the _option suffix a convention we're starting, already using?  where does this come from?

Reply:
It's new. Traditionally we've omitted any distinction between `Option<T>` and `T`. If we're going to distinguish these, I'd prefer someone poke around for a community convention first.

Reply:
my personal preference would be to not make a distinction